frontend/src/pages/_app.tsx

```diff
@@ -157,17 +157,6 @@ const App: AppType = ({
 };
 
 App.getInitialProps = async (appContext) => {
-  // WARNING: Do not remove this getInitialProps implementation as it
```

Comment:
Do we actually need that comment here? It doesn't seem so

Reply:
Well, it used to be a thin proxy-function implemented here, but later i18n brought some use for this method to exist, and indeed, this comment can be dropped now.